### DISCUSSION TITLE
CSS for left side nav headers/titles

### DISCUSSION THREAD

**Discussion by ledgley** [squidfunk/mkdocs-material#4211] [Q&A]
I have found I can customise the style of the title of the ToC on the right side of the page with this
```
.md-nav__title {
  font-weight: lighter;
  color: #004165;
  font-size: 0.95rem;
}
```
I would also like to customise the style of the title and section headers of the nav on the left side of the page. I can't find any CSS to specifically alter the title and section headers, I can only modify the entire nav style.

Is this possible?

**Comment by squidfunk** [comment]
Yes, it is possible to customize the entirety of Material for MkDocs and you're likely having troubles with [specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity). I'm afraid I'm currently lacking the time to provide a solution, but maybe somebody from the community can help.

**Comment by fertek** [comment]
> I can't find any CSS to specifically alter the title and section headers

Try:

```
/* left navigation: title */
.md-nav--primary .md-nav__title {
    background: #cff;
}

/* left navigation: section headers */
.md-sidebar--primary .md-nav__item--section > label.md-nav__link {
    background: #cfc;    	 	
}
```
